pkgs/by-name/op/openmpi/package.nix

```diff
@@ -96,6 +95,7 @@ stdenv.mkDerivation (finalAttrs: {
       ucc
     ]
     ++ lib.optionals cudaSupport [ cudaPackages.cuda_cudart ]
+    ++ lib.optionals enablePRRTE [ prrte ]
```

Comment:
This should be part of the changes to the `openmpi` changes in the previous commit.